ex1/myScript.js

```diff
@@ -0,0 +1,3 @@
+function myFunction() {
```

Comment:
Please change to a meaningful and descriptive name and never ever call functions or variables names that have mo meaning